GDJS/Runtime/runtimebehavior.js

```diff
@@ -5,9 +5,9 @@
  */
 
 /**
- * @typedef BehaviorData Properties to set up a behavior.
+ * @typedef BehaviorData The behavior type. Used by GDJS to find the proper behavior to construct.
```

Comment:
Be careful seems that you do not started your work from `master` so you have commits that are undoing what was previously fixed.